go/consensus/cometbft/apps/keymanager/churp/txs_test.go

```diff
@@ -205,6 +215,26 @@ func (s *TxTestSuite) TestCreate() {
 		require.ErrorContains(s.T(), err, "invalid config: unsupported group, ID 100")
 	})
 
+	s.Run("not enough stake", func() {
```

Comment:
Also add a test with sufficient stake and check that stake claims are created correctly.